src/server/dflycmd.h

```diff
@@ -46,7 +46,7 @@ struct FlowInfo {
   DflyVersion version;
 
   std::optional<LSN> start_partial_sync_at;
-  uint64_t last_acked_lsn;
+  uint64_t last_acked_lsn = 0;
```

Comment:
If it's possible to use this class before initialization, shouldn't we also initialize `conn` and `version`?